tests/trestle/tasks/tanium_to_oscal_test.py

```diff
@@ -123,16 +158,46 @@ def test_tanium_simulate_bad_input_file(tmp_path):
     assert len(os.listdir(str(tmp_path))) == 0
 
 
-@patch(target='uuid.uuid4', new=uuid_mock1)
-def test_tanium_execute(tmp_path):
+def test_tanium_execute(tmp_path, monkeypatch: MonkeyPatch):
+    """Test execute call."""
+    monkeybusiness = MonkeyBusiness()
+    monkeypatch.setattr(tanium_helper, '_uuid_component', monkeybusiness.uuid_component)
+    monkeypatch.setattr(tanium_helper, '_uuid_inventory', monkeybusiness.uuid_inventory)
+    monkeypatch.setattr(tanium_helper, '_uuid_observation', monkeybusiness.uuid_observation)
+    monkeypatch.setattr(tanium_helper, '_uuid_result', monkeybusiness.uuid_result)
+    tanium.TaniumTransformer.set_timestamp('2021-02-24T19:31:13+00:00')
+    assert tanium.TaniumTransformer.get_timestamp() == '2021-02-24T19:31:13+00:00'
+    config = configparser.ConfigParser()
+    config_path = pathlib.Path('tests/data/tasks/tanium/demo-tanium-to-oscal.config')
+    config.read(config_path)
+    section = config['task.tanium-to-oscal']
+    section['output-dir'] = str(tmp_path)
+    section['cpus-max'] = '1'
+    tgt = tanium_to_oscal.TaniumToOscal(section)
+    retval = tgt.execute()
+    assert retval == TaskOutcome.SUCCESS
+    assert len(os.listdir(str(tmp_path))) == 1
+    f_expected = pathlib.Path('tests/data/tasks/tanium/output/') / 'Tanium.oscal.json'
+    f_produced = tmp_path / 'Tanium.oscal.json'
```

Comment:
This looks a little odd so I would make a string for the json file name

Reply:
Not sure which is the odd part? This shows that there are 2 files named the same in different locations that need to be compared. The 1st is relative to the test itself and the 2nd is relative to the temp output dir. Please give an example of what is preferable.

Reply:
This is very minor but in line 180 you could have put  the Tanium string directly into the Path string rather than adding it on separately.  Since the Tanium string is repeated twice I would define a string for tanium_filename and then use it on those two lines.  Very minor suggestion.

Reply:
Funny. Some time ago when contributing to Auditree, I got a comment from Al that said I should not "define a string for tanium_filename and then use it on those two lines". I'll leave it be for now.